master/contrib/github_buildbot.py

```diff
@@ -143,7 +144,27 @@ def render_POST(self, request):
             request.setResponseCode(INTERNAL_SERVER_ERROR)
             return json.dumps({"error": e.message})
 
-    def process_change(self, payload, user, repo, repo_url, project, request):
+    def process_change(self, change, branch, repo, repo_url):
+        files = change['added'] + change['removed'] + change['modified']
+        who = ""
+        if 'username' in change['author']:
```

Comment:
Side note: we seem to have a number of places where similar code is present, it feels to be worth to be extracted to function...